hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md

```diff
@@ -129,8 +129,6 @@ Rebuild your Docker image.
 
 </CodeGroup>
 
----
```

Comment:
I have these lines to identify the non-code text that is inside the code group items. As the text changes, it may not be apparent to the reader where the code group item ends.
I am adding them back.